Samples/Hast.Samples.Consumer/Program.cs

```diff
@@ -212,7 +209,9 @@ string GetArgument(string name)
 
             Console.WriteLine("Starting hardware execution.");
 
-            // Running samples.
+            // Running the selected sample. It is executed 3 times to ensure all JIT compilation overhead has been

```

Comment:
```suggestion
            // Running the selected sample. They're executed 3 times to ensure all JIT compilation overhead has been

```

Reply:
Szerintem nem. Egy példát ("the selected sample") futtatsz három alkalommal, ezért egyes számú az alany.

Reply:
Ja tényleg.